test/widgets/subscription_list_test.dart

```diff
@@ -190,4 +190,41 @@ void main() {
     check(tester.widget<UnreadCountBadge>(find.byType(UnreadCountBadge)).backgroundColor)
       .equals(swatch);
   });
+
+  testWidgets('muted streams are displayed as faded', (tester) async {
+    final stream = eg.stream(name: 'Channel 1');
```

Comment:
```suggestion
    final stream1 = eg.stream(streamId: 1, name: 'Channel 1');
```
(nit): Let's numerise this stream name as well as add an explicit `streamId` to make it match `stream2`.